registry/reusables/ui/notify/notify-provider.tsx

```
@@ -133,364 +57,95 @@ export function ToastProvider({
+        <div
+          key={position}
+          className={cn(
+            toastPositionVariants({
+              position: position as ToastPosition,
+            }),
+            classNames.containerClassName
+          )}
+        >
+          <AnimatePresence mode="popLayout">
+            {positionToasts.map((toast) => {
+              const { id, dismiss: dismissToast, params, paused } = toast
```

ESLint

[error] 'dismissToast' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)